project-changed.py

```diff
@@ -2,10 +2,9 @@
 import subprocess
 
 if len(sys.argv) != 3:
-    sys.stderr.write(f'''usage: {sys.argv[0]} <deployed-hash> <project>
+    sys.stderr.write(f'''usage: {sys.argv[0]} <orig-hash> <project>
```

Comment:
this script would be simpler in bash, probably. But fine like this.

Reply:
Yeah, maybe.  I think my main motivation for Python is that I want to stick the project information in a JSON file and include dependencies (e.g. we the typecheck module a toplevel project and use it in all the Python projects) and that seemed like it would be easier to work with in Python.